### PR TITLE
service config: add WRR LB policy config

### PR DESCRIPTION
**Pull request by markdroth**

### CODE REVIEW

File: grpc/service_config/service_config.proto [hunk 1]
```diff
@@ -196,6 +196,21 @@ message PickFirstConfig {}
 // Configuration for round_robin LB policy.
 message RoundRobinConfig {}
 
+// Configuration for weighted_round_robin LB policy.
```
Comment (yousukseung) [inline grpc/service_config/service_config.proto:199]: For the same reason we prefixed it with "client_side" in Envoy, can we mention it's 'client-side' in the comment here?

Comment (markdroth) [reply grpc/service_config/service_config.proto:199]: Not necessary in this case, because this is the only WRR policy that gRPC plans to support.

**Review by yousukseung — APPROVED** [review]

File: grpc/service_config/service_config.proto [hunk 2]
```diff
@@ -202,13 +202,18 @@ message WeightedRoundRobinLbConfig {
   // the endpoints. By default, per-request utilization reporting is used.
   google.protobuf.BoolValue enable_oob_load_report = 1;
 
+  // Load reporting interval to request from the server.  Note that the
+  // server may not provide reports as frequently as the client requests.
+  // Used only when enable_oob_load_report is true.
```
Comment (yousukseung) [inline grpc/service_config/service_config.proto]: "Defaults to 10 seconds."?

Comment (markdroth) [reply grpc/service_config/service_config.proto]: Yup, already did that -- you must not have been looking at the latest commit.

**Pull request merged**